src/scripts.js

```diff
@@ -38,16 +38,17 @@ logo.addEventListener('click', goHome);
 searchBar.addEventListener('keyup', searchRecipes);
 
 //functions
-apiCalls().then(data => {
-  users = data[0].usersData;
-  ingredients = data[1].ingredientsData;
-  recipes = data[2].recipeData;
+apiObject.getAllPromises().then(data => {
+  users = data[0];
```

Comment:
Did we want to leave the user as hardcoded?

Reply:
Good catch - I'm not sure it matters either way. Maybe we can revert in our last merge?

Reply:
works for me!